kernel_gateway/services/activity/handlers.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+import tornado.web
+import json
+from .manager import activity
+from ...mixins import TokenAuthorizationMixin, CORSMixin
+
+class ActivityHandler(TokenAuthorizationMixin, CORSMixin, tornado.web.RequestHandler):
+
+    def get(self, **kwargs):
```

Comment:
Decorate this with `@json_errors`.

https://github.com/jupyter-incubator/kernel_gateway/blob/master/kernel_gateway/services/kernels/handlers.py#L27